Add header where various compat code is going to be stored.

Index: src/drivers/net/ethernet/intel/i40e/i40e.h
===================================================================
--- src.orig/drivers/net/ethernet/intel/i40e/i40e.h	2017-04-27 17:45:31.462756757 +0200
+++ src/drivers/net/ethernet/intel/i40e/i40e.h	2017-04-27 19:24:14.013005943 +0200
@@ -65,6 +65,8 @@
 #include "i40e_txrx.h"
 #include "i40e_dcb.h"
 
+#include "i40e_backport_compat.h"
+
 /* Useful i40e defaults */
 #define I40E_MAX_VEB			16
 
Index: src/drivers/net/ethernet/intel/i40e/i40e_backport_compat.h
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ src/drivers/net/ethernet/intel/i40e/i40e_backport_compat.h	2017-04-27 19:21:08.333216409 +0200
@@ -0,0 +1,4 @@
+#ifndef _RH_I40E_BACKPORT_COMPAT_H_
+#define _RH_I40E_BACKPORT_COMPAT_H_
+
+#endif /* #ifndef _RH_I40E_BACKPORT_COMPAT_H_ */
